### ISSUE TITLE
Datadog metrics are missing relational information

### ISSUE THREAD

**Issue by amolari** [F5Networks/f5-telemetry-streaming#170]
### Environment
 * Telemetry Streaming Version: 1.23
 * BIG-IP Version: 14.1.4

### Summary
Cannot create a relation between pool member and its pool.

We use the Datadog consumer (I believe this issue is not consumer-specific).
In the consumer trace, I see

```
                   {
                        "host": "ip-10-100-2-55.eu-west-1.compute.internal",
                        "interval": 60,
                        "metric": "f5_test_another.pools.members.serverside.bitsOut",
                        "points": [
                            [
                                1632493905.325,
                                0
                            ]
                        ],
                        "tags": [
                            "addr:10.200.1.52",
                            "monitorStatus:up",
                            "port:443",
                            "availabilityState:available",
                            "enabledState:enabled",
                            "status.statusReason:Pool member is available"
                        ],
                        "type": "gauge"
                    },

```
We need to link this information with the pool. A tag such as `pool_name` is missing.

**Comment by amolari** [comment]
@dstokesf5 `f5_test_another` is a metric prefix I'm adding to all metrics (code customization in the meantime https://github.com/F5Networks/f5-telemetry-streaming/issues/158 gets eventually implemented).
The pool name is not showing. As far as I understand, it's (poolName) excluded from the normalization (https://github.com/F5Networks/f5-telemetry-streaming/blob/e9774dbd6118323c7215bcb762f2878451402a9d/src/lib/properties.json#L805-L824)

**Comment by dstokesf5** [comment]
Thank you for your feedback. I have added this issue to our internal product backlog as AUTOTOOL-2800.

**Comment by amolari** [comment]
I guess it had been fixed in v1.25.0?

**Comment by dstokesf5** [comment]
This has been resolved in the 1.25.0 release.
